guidelines-for-respectful-communication/index.html

```diff
@@ -12,54 +12,117 @@ <h1 class="text-2xl-5xl">Haskell Foundation Guidelines For Respectful Communicat
   <div class="mx-auto prose md:prose-lg">
 
     <h2>Preamble</h2>
-    <p>The Haskell Foundation has adopted these guidelines for respectful communication.</p>
 
-    <p>We do not seek to impose these guidelines on members of the Haskell community generally. Rather it is a signal that we seek high standards of discourse in the Haskell community, and are willing to publicly hold ourselves to that standard, in the hope that others may voluntarily follow suit.</p>
+    <p>The Haskell Foundation has adopted the Standards of Public Behaviour set
+    out below. As representatives of the Foundation, we commit ourselves to
+    those standards and we aspire to apply them in all our public interactions
```

Comment:
This could use an explicit clarification whether we talk about HF-related interactions only or any Haskell-related at all.

Reply:
Agreed. I'll raise a further commit with a concrete suggestion on that specific point, for discussion.